### DISCUSSION TITLE
QZ Forerunner35 HR

### DISCUSSION THREAD

**Discussion by jorgehuedo** [cagnulein/qdomyos-zwift#1667] [Q&A]
Hi all,
Sorry if it's already solved, but I have been searching and I'm a bit lost.

I have a Forerunner35 and I would like to connect it to my QZ to see my HR as well as data of my bike.
Garmin Connect is connected in the same phone than QZ (my phone doesn't have ANT+).
I can see my HR on Garmin Connect but QZ is always 0.

Is this a possible setup? Or I'm missing something?
Thanks.

**Comment by cagnulein** [comment]
@jorgehuedo did you download the qz companion app on the watch?

**Comment by jorgehuedo** [comment]
No, my watch is not compatible with Apps. Is there any other way?

**Comment by cagnulein** [comment]
you need to buy an usb ant dongle to connect to your phone :)
